packages/AI/VectorsPinecone/src/models/PineconeDatabase.ts

```diff
@@ -185,12 +185,12 @@ export class PineconeDatabase extends VectorDBBase {
 
                 let duplicate: PotentialDuplicate = new PotentialDuplicate();
                 duplicate.ProbabilityScore = record.score;
-                duplicate.PrimaryKeyValues = metadata.PrimaryKeys.map((pk: string) => {
+                duplicate.KeyValuePairs = metadata.PrimaryKeys.map((pk: string) => {
```

Comment:
@JS-BC  again - if you clean up here you should not be using KeyValuePairs, why would you do that when you have CompositeKey now??